ops/gulp/pkg-css.js

```diff
@@ -5,117 +5,91 @@ const gulpif = require('gulp-if');
 const sourcemaps = require('gulp-sourcemaps');
 const cleanCSS = require('gulp-clean-css');
 const autoprefixer = require('gulp-autoprefixer');
-const debug = require('gulp-debug');
-const gulpSequence = require('gulp-sequence');
-const replace = require('gulp-replace');
 const header = require('gulp-header');
 const config = require('../config');
 const pkg = require('../../package');
-
-let environment = require('../lib/environment');
-
+const environment = require('../lib/environment');
+const signale = require('signale');
 const paths = {
-	src: config.root.css,
-	dest: environment.production ? config.root.dest : config.root.tmp
+    src: config.root.css,
+    dest: environment.production ? config.root.dest : config.root.tmp
 }
 
-var d = new Date();
-var y = d.getFullYear();
-var banner = `/*!
+const d = new Date();
+const y = d.getFullYear();
+const banner = `/*!
 * Fiori Fundamentals v${pkg.version}
 * Copyright (c) ${y} SAP SE or an SAP affiliate company.
 * Licensed under Apache License 2.0 (https://github.com/SAP/Fundamental/blob/master/LICENSE)
 */\n`;
 
 //compile top-level files
-var sassTask = (cb) => {
-    let prefix = config.tasks.css.prefix;
-    let files = environment.production ? `${paths.src}/*.${config.tasks.css.extensions}` : `!${paths.src}/all.scss`;
```

Comment:
I still don't understand how this was working to be honest. Maybe it wasn't? Anyway in Gulp 4 it was correctly throwing a ```Missing positive glob``` error.